src/clarity-angular/tabs/vtabs.ts

```diff
@@ -0,0 +1,212 @@
+            testLink.activated = true;
+
+            let testContent: VTabsContent = this.vtabContents.first;
+            testContent.activated = true;
+        } else {
+            this.activateTabContentAtIndex(this.activatedLink);
+        }
+    }
+
+    /**
+     * @function updateForActiveLink
+     *
+     * @description
+     * Method to update all the VTabNavLinks when an activatedChange event is heard by:
+     *
+     * 1. Deactivating all the linkItem.ids that don't match the updatedLink passed as arg
+     * 2. Identifying the index of the newly activated tab link so we can activate corresponding VTabContent
+     * 3. Activating the corresponding VTabsContent item.
+     *
+     * @param updatedLink -  a VTabNavLink that was clicked on.
+     *
+     */
+    updateForActiveLink(updatedLink: VTabsNavLink) {
+        let linkIndex: number; // Map between link and content
+        this.vtabLinks.forEach((linkItem: VTabsNavLink, index: number) => {
+            if (linkItem.id !== updatedLink.id) {
+                linkItem.activated = false;
+            } else {
+                linkIndex = index; // Position of the link that was clicked.
+                this.activatedLink = index;
+            }
+        });
+
+        // Update content for index
+        if (typeof linkIndex !== "undefined") { // Remember, remember 0 is also false.
+            this.activateTabContentAtIndex(linkIndex);
+        }
+
+        // Broadcast the link change event
+        this.linkChange.emit(updatedLink);
+    }
+
+    /**
+     * @function activateTabContentAtIndex
+     *
+     * @description
+     * Method to update VTabContent items as follows:
+     *
+     * 1. Deactivate all VTabContent items not equal to the linkIndex arg
+     * 2. Activate the VTabContent iotem that has the same index as the linkIndex arg
+     *
+     * @param linkIndex - the index to activate in this.vtabsContent QueryList
+     */
+    activateTabContentAtIndex(linkIndex: number) {
+        this.vtabContents.forEach((content, contentIndex) => {
+            if (contentIndex === linkIndex) {
+                content.activated = true;
+                this.contentChange.emit(content); // Broadcast the content that was activated.
+            } else {
+                content.activated = false;
+            }
+        });
+    }
+
+    /**
+     * @function mapContentId
+     *
+     * @description
+     * Method that sets VTabContent items to the corresponding VTabNavLink item.
+     * Called during the ngAfterContentInit method when iterating over the QueryList of VTabNavLinks
+     *
+     * @param linkIndex - the index of the link calling this method
+     * @param id - the id string to set for the matching VTabContent item
+     */
+    mapContentId(linkIndex: number, id: string) {
+        this.vtabContents.forEach((content, contentIndex) => {
+            if (linkIndex === contentIndex) {
+                content.id = id;
+            }
+        });
+    }
```

Comment:
I have wizard on the brain too much. The _old_ wizard had added skip/unskip tab functionality. It's not part of tabs.

Feel free to ignore this comment. My bad.